revolv/payments/models.py

```diff
@@ -23,63 +36,92 @@ class PaymentInstrumentType(models.Model):
     objects = PaymentInstrumentTypeManager()
 
 
-class PaymentTransactionManager(models.Manager):
+class PaymentManager(models.Manager):
+    """
+        Simple manager for the Payment model.
+    """
+
+    def payments(self, user=None, project=None, queryset=None):
+        """
+        :return: Payments associated with this user
+        """
+        if queryset is None:
+            queryset = super(PaymentManager, self).get_queryset()
+        payments = queryset.filter(user=user).order_by('created_at')
+        return payments
+
+    def donations(self, user=None, project=None, queryset=None):
```

Comment:
He just means all the methods of `PaymentManager`. Tbh I think @SamLau95 is right, this code could be dried up with a a delegation function